### PR TITLE
Upgrade config and organize imports

### PR DESCRIPTION
**Pull request by forabi**
Use VS Code's new "Organize Imports" feature and apply it on document save.

### CODE REVIEW

File: clownOverrides/c__package.json [hunk 1]
```diff
@@ -3,13 +3,6 @@
...
     "pre-push": "yarn pretest",
-    "lint-ts/tslint":
-      "tslint '**/*.ts{,x}' --project .  --exclude '**/node_modules/**' --exclude '**/.serverless/**'  --exclude '**/.webpack/**'  --exclude '**/dist/**' --exclude 'src/typings/schema.ts' --exclude 'clients/**/*' --exclude 'src/database/migrations/*.ts'"
-  },
-  "devDependencies": {
-    "tslint": "^5.11.0",
-    "ts-jest": "^23.0.1",
-    "typescript": "^3.0.3",
-    "typescript-eslint-parser": "^18.0.0"
+    "lint-ts/tslint": "tslint '**/*.ts{,x}' --project .  --exclude '**/node_modules/**' --exclude '**/.serverless/**'  --exclude '**/.webpack/**'  --exclude '**/dist/**' --exclude 'src/typings/schema.ts' --exclude 'clients/**/*' --exclude 'src/database/migrations/*.ts'"
```
Comment (wholesomedev) [inline clownOverrides/c__package.json:6]: Nit: some characters are separated by two spaces

**Review by wholesomedev — COMMENTED** [review]
Very cool to have auto organize imports feature. Although, I feel the VS Code team should've made that into, like, an ESLint plugin. That would've made things better.

**Comment by forabi** [comment]
> Very cool to have auto organize imports feature. Although, I feel the VS Code team should've made that into, like, an ESLint plugin. That would've made things better.

There's actually an ESLint rule which we currently use but it would be really tedious to have to sort the imports manually to suppress the error. I don't know if the rule has an automatic fix but it most probably won't work with TS files.

File: clownOverrides/c__package.json (hunk 1 above)
Comment (forabi) [reply clownOverrides/c__package.json:6]: Whoa! Nice catch! :smile:

**Pull request merged**